sway/layout.c

```diff
@@ -769,21 +808,42 @@ void update_geometry(swayc_t *container) {
 	}
 }
 
+/**
+ * Layout application prototypes
+ */
+static void apply_horiz_layout(swayc_t *container, const double x,
+			       const double y, const double width,
+			       const double height, const int start,
+			       const int end);
+static void apply_vert_layout(swayc_t *container, const double x,
+			      const double y, const double width,
+			      const double height, const int start,
+			      const int end);
+static void apply_tabbed_or_stacked_layout(swayc_t *container, double x,
+					   double y, double width,
+					   double height);
+
+static void apply_auto_layout(swayc_t *container, const double x, const double y,
+			      const double width, const double height,
+			      enum swayc_layouts group_layout,
+			      bool master_first);
```

Comment:
grepped through the code for mixed tab/space indents and removed the ones I had put in.